attributes/default.rb

```diff
@@ -1,112 +1,83 @@
-# Enable default repo
-default['mesos']['repo']    = true
+# Default Java version
+default['java']['jdk_version'] = '7'
+
+# Use Mesosphere repo
+default['mesos']['repo']       = true
 
 # Mesosphere Mesos version.
-default['mesos']['version'] = '0.22.1'
+default['mesos']['version']    = '0.25.0'
 
 # Init system to use
-default['mesos']['init']    = case node['platform']
-                              when 'debian' then 'sysvinit_debian'
-                              when 'centos', 'amazon', 'scientific'
-                                node['platform_version'].to_i >= 7 ? 'systemd' : 'upstart'
-                              else 'upstart'
-                              end
+default['mesos']['init']       = case node['platform']
+                                 when 'debian'
+                                   node['platform_version'].to_i >= 8 ? 'systemd' : 'sysvinit_debian'
+                                 when 'ubuntu'
+                                   node['platform_version'].to_f >= 15.04 ? 'systemd' : 'upstart'
+                                 when 'redhat', 'centos', 'amazon', 'scientific'
+                                   node['platform_version'].to_i >= 7 ? 'systemd' : 'upstart'
+                                 else 'upstart'
+                                 end
 
 #
 # Mesos MASTER configuration
 #
 
-# Mesos master binary location
-if node['mesos']['version'] == '0.21.1' || node['mesos']['version'] == '0.22.0' || node['mesos']['version'] == '0.22.1' || node['mesos']['version'] == '0.23.0'
-  default['mesos']['master']['bin']                         = '/usr/sbin/mesos-master'
-else
-  default['mesos']['master']['bin']                         = '/usr/local/sbin/mesos-master'
```

Comment:
you should mention in the release notes that older version of mesos support has been dropped